cfme/tests/ansible/test_embedded_ansible_crud.py

```diff
@@ -82,3 +87,62 @@ def is_ansible_pod_stopped(enabled_embedded_appliance):
         assert wait_for(is_nginx_stopped, func_args=[enabled_embedded_appliance], num_sec=30)
     else:
         assert wait_for(is_ansible_pod_stopped, func_args=[enabled_embedded_appliance], num_sec=180)
+
+
+@pytest.mark.tier(1)
+def test_embedded_ansible_event_catcher_process(appliance, enabled_embedded_appliance):
```

Comment:
I think its redundant to use both fixtures here, when `enabled_embedded_appliance` yields the appliance object.

I think this should only take `enabled_embedded_appliance`, and should use it directly.

Reply:
Updating PR with suggested changes.

Reply:
Updated.